src/main/ruby/core/enumerator.rb

```diff
@@ -541,11 +541,3 @@ def reset
     end
   end
 end
-
-module Kernel
-  def to_enum(method=:each, *args)
-    Enumerator.new(self, method, *args)
-  end
-
-  alias_method :enum_for, :to_enum
-end
```

Comment:
Nice catch, it's already defined in kernel.rb.
Maybe we should implement method redefinition warnings like MRI to detect these easily:
```
multidef.rb:4: warning: method redefined; discarding old abc
multidef.rb:1: warning: previous definition of abc was here
```